kotest-property/src/commonMain/kotlin/io/kotest/property/Gen.kt

```diff
@@ -82,6 +74,19 @@ abstract class Arb<out A> : Gen<A>() {
    @Deprecated("implement one value at a time using value(rs)", ReplaceWith("value(rs)"))
    open fun values(rs: RandomSource): Sequence<Sample<A>> = emptySequence()
 
+   /**
+    * Returns a sequence from values generated from this arb.
+    * Edgecases will be ignored.
+    */
+   fun randomSamples(rs: RandomSource = RandomSource.Default): Sequence<Sample<A>> {
```

Comment:
random is kinda implied by the fact this is an arb and you pass in the rs.
Maybe on the next pr rename to just samples ?

Reply:
awesome - come to think of it, even `value(rs)` really is `sample(rs: RandomSeed): Sample<A>`. I can rename that too in the subsequent PR if that makes more sense.